test/unit/ui/map/isMoving.test.js

```diff
@@ -39,12 +39,18 @@ test('Map#isMoving returns true during a camera zoom animation', (t) => {
 test('Map#isMoving returns true when drag panning', (t) => {
     const map = createMap(t);
 
+    map.on('movestart', () => {
```

Comment:
Do you know if we have enough coverage around whether those events are triggered under a similar context?

Reply:
This test case is focused around `isMoving`. But what I'm thinking is covering whether we call the lambda function at all, if we fail to call that lambda it would go unnoticed.

Maybe collecting how many times those lambdas are called and testing that at the end of this block could be useful.

Reply:
I think in this case there is enough coverage elsewhere but we could change the test to avoid this problem.